src/engine/strat_engine/metadata.rs

```diff
@@ -560,7 +560,7 @@ mod mda {
 
                     Ok(Some(MDAHeader {
                                 used: used,
-                                last_updated: Timespec::new(secs as i64, nsecs as i32),
+                                last_updated: Local.timestamp(secs as i64, nsecs),
```

Comment:
Because of this change, you can also omit assert about nsecs above. Maybe then you can get rid of ```nsecs``` name entirely and just use the expression here.